assets/state-kata-manager/0600_daemonset.yaml

```diff
@@ -66,23 +66,23 @@ spec:
           readinessProbe:
             exec:
               command:
-                - bash
+                - sh
                 - "-c"
                 - |
                   pid_file="${KATA_ARTIFACTS_DIR}/k8s-kata-manager.pid"
-                  [[ -f "${pid_file}" ]] && [[ -d "/proc/$(cat $pid_file)" ]] && exit 0 || exit 1
+                  [ -f "${pid_file}" ] && [ -d "/proc/$(cat $pid_file)" ] && exit 0 || exit 1
             initialDelaySeconds: 5
             periodSeconds: 5
       terminationGracePeriodSeconds: 30
       volumes:
         - name: nvidia-kata-manager-entrypoint
           configMap:
             name: nvidia-kata-manager-entrypoint
-            defaultMode: 448
+            defaultMode: 493
```

Comment:
I don't believe this change should be needed if the `k8s-kata-manager` is running with uid 0 and gid 0. Can you please verify?